execute/plugin_test.go

```diff
@@ -166,3 +155,496 @@ func Test_getPendingExecutedReports(t *testing.T) {
 		})
 	}
 }
+
+// TODO: better than this
+type tdr struct{}
+
+func (t tdr) ReadTokenData(
+	ctx context.Context, srcChain cciptypes.ChainSelector, num cciptypes.SeqNum) ([][]byte, error,
+) {
+	return nil, nil
+}
+
+// mustRandID generates a random hex ID value.
+func mustRandID() string {
+	bytes := make([]byte, 32)
+	if _, err := rand.Read(bytes); err != nil {
+		panic(err)
+	}
+	return hex.EncodeToString(bytes)[:32]
+}
+
+func breakCommitReport(
+	commitReport cciptypes.ExecutePluginCommitDataWithMessages,
+) cciptypes.ExecutePluginCommitDataWithMessages {
+	commitReport.Messages = append(commitReport.Messages, cciptypes.CCIPMsg{
+		CCIPMsgBaseDetails: cciptypes.CCIPMsgBaseDetails{
+			ID:          mustRandID(),
+			SourceChain: cciptypes.ChainSelector(1),
+			SeqNum:      cciptypes.SeqNum(999),
+			MsgHash:     cciptypes.Bytes32{},
+		},
+	})
+	return commitReport
+}
+
+func makeTestCommitReport(
+	numMessages, srcChain, firstSeqNum, block int, timestamp int64, executed []cciptypes.SeqNum,
+) cciptypes.ExecutePluginCommitDataWithMessages {
+	for _, e := range executed {
+		if e < cciptypes.SeqNum(firstSeqNum) || e > cciptypes.SeqNum(firstSeqNum+numMessages-1) {
+			panic("executed message out of range")
```

Comment:
nit: I wonder if its nicer for test output to have this be in a `require.True` instead of doing a `panic`, e.g

```golang
inRange := e >= cciptypes.SeqNum(firstSeqNum) && e <= cciptypes.SeqNum(firstSeqNum+numMessages-1)
require.True(t, inRange)
```

Reply:
Maybe. I went with a panic since this is only used when setting up the test cases.